cmd/create/service/cmd.go

```diff
@@ -401,7 +400,8 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 
 	// operator role logic.
-	operatorRolesPrefix := getRolePrefix(args.ClusterName)
+	clusterName := strings.Trim(args.ClusterName, " \t")
```

Comment:
Why do we need to trim the cluster name?

Reply:
This was already done on create/cluster but not on create/service. It is just to keep things closer since they do really close flows